### DISCUSSION TITLE
When building an Android app with AAR, I encounter dependency issues. How can I solve this?

### DISCUSSION THREAD

**Discussion by Amazonzero** [maplibre/maplibre-native#2332] [Q&A]
![image](https://github.com/maplibre/maplibre-native/assets/12524173/a83d1961-dd6c-41df-b4a6-4dbcbbc42162)
like this ,i saw this，Is there any step I'm missing?，i'm use this
![image](https://github.com/maplibre/maplibre-native/assets/12524173/366aa916-f824-4e3a-9dd0-b84a10128032)

![image](https://github.com/maplibre/maplibre-native/assets/12524173/2ff5d328-ad90-4e07-8621-e66fdfcb1b6b)

**Comment by louwers** [comment]
Can you share the entire log as text?

**Reply by Amazonzero** [reply]
[log.txt](https://github.com/maplibre/maplibre-native/files/15123650/log.txt)
Thanks louwes . I'm looking forward to helping to resolve this issue

**Reply by louwers** [reply]
Could you try using the version published to Maven Central?

I'll look into the `.aar`.
